pkg/image/apiserver/apiserver.go

```diff
@@ -165,10 +175,13 @@ func (c *completedConfig) newV1RESTStorage() (map[string]rest.Storage, error) {
 		whitelister = whitelist.WhitelistAllRegistries()
 	}
 
+	imageLayerIndex := imagestreametcd.NewImageLayerIndex(imageStorage)
```

Comment:
I'd prefer to wire this off of the loopback client instead of storage.  Doing so would eliminate the closure, side-effect, call me after A and before C from different call site behavior.